Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.6
```

Comment:
use a vanilla docker image as your base image.  Its more reliable and more secure.